.github/workflows/ci.yml

```diff
@@ -737,6 +737,36 @@ jobs:
           eval "$(./uv generate-shell-completion bash)"
           eval "$(./uvx --generate-shell-completion bash)"
 
+  smoke-test-linux-musl:
+    timeout-minutes: 10
+    needs: build-binary-linux-musl
+    name: "check system | alpine"
+    runs-on: ubuntu-latest
+    container: alpine:latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Download binary"
+        uses: actions/download-artifact@v4
+        with:
+          name: uv-linux-musl-${{ github.sha }}
+
+      - name: "Prepare binary"
+        run: |
+          chmod +x ./uv
+          chmod +x ./uvx
+
+      - name: "Smoke test"
+        run: |
+          # Overwrite the 3.13.0 pin from the project, there are not functional
+          # musl distributions for it
+          ./uv python pin 3.13
```

Comment:
Could this potentially affect real-world users?

Reply:
Yes, but it'll just say "No Python download found for request" (as we do for non-musl distributions) instead of the special case "uv does not support musl downloads yet"